extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -444,6 +444,9 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 				{
 					'sessionId': this._sessionId,
 					'performanceDataSource': this._skuRecommendationPerformanceDataSource,
+					'scalingFactor': this._skuScalingFactor?.toString(),
```

Comment:
@raymondtruong not sure if we should club these parameters with 'instancerequirement' event or 'getskurecommendation' event. Thoughts?

Reply:
I think putting it here (GetInstanceRequirements) makes more sense. Having it with GetSkuRecommendation wouldn't really help if you have the input parameters and the outputted SKU but without the intermediate requirements object. Ideally we'd have all 3 of them (input parameters, requirements object, and final recommendation result) together so that we can see it all in one place, like we discussed this morning. Do you think we should combine them?

Reply:
Discussed internally, going with adding 'input parameters' to 'GetSkuRecommendation' event.

Reply:
@Charles-Gagnon for visibility.
@raymondtruong please take a look.